references/cli/status.go

```diff
@@ -466,6 +474,22 @@ func printApplicationTree(c common.Args, cmd *cobra.Command, appName string, app
 	return nil
 }
 
+func launchUI(c common.Args, _ *cobra.Command) error {
+	k8sClient, err := c.GetClient()
+	if err != nil {
+		return fmt.Errorf("cannot get k8s client: %w", err)
+	}
+	restConfig, err := c.GetConfig()
+	if err != nil {
+		return errors.Wrapf(err, "failed to get kube config, You can set KUBECONFIG env or make file ~/.kube/config")
```

Comment:
`make file ~/.kube/config` is not valid path for both Windows, Mac and Linux

Reply:
ok, I have fix it.